x/ccv/consumer/keeper/validators.go

```diff
@@ -97,18 +139,54 @@ func (k Keeper) Slash(ctx sdk.Context, addr sdk.ConsAddress, infractionHeight, p
 }
 
 // Jail - unimplemented on CCV keeper
```

Comment:
We'll need to remove `// Jail - unimplemented on CCV keeper`, these methods are now implemented

Reply:
Okay